src/Grid/grid.jl

```diff
@@ -47,6 +47,8 @@ getrefshape(::AbstractCell{refshape}) where refshape = refshape
 nvertices(c::AbstractCell) = length(vertices(c))
 nedges(   c::AbstractCell) = length(edges(c))
 nfaces(   c::AbstractCell) = length(faces(c))
+nvertices(::Type{T}) where {T <: AbstractRefShape} = length(reference_vertices(T))
+nedges(   ::Type{T}) where {T <: AbstractRefShape} = length(reference_edges(T))
```

Comment:
seems to be missing in the tests according to codecov

Reply:
Solved (with nice redundant code reduction too :))